src/sentry/static/sentry/app/components/forms/apiForm.jsx

```diff
@@ -11,7 +11,13 @@ export default class ApiForm extends Form {
     ...Form.propTypes,
     onSubmit: PropTypes.func,
     apiMethod: PropTypes.string.isRequired,
-    apiEndpoint: PropTypes.string.isRequired
+    apiEndpoint: PropTypes.string.isRequired,
+    disabledFields: PropTypes.array,
```

Comment:
Let's move this to `forms/form.jsx`.

Can we rename it maybe... `excludeFromSubmit` since without context "disabled" sounds more like a description of UI state?